oteapi/strategies/filter/crop_filter.py

```diff
@@ -1,35 +1,41 @@
 """Demo-filter strategy"""
-# pylint: disable=unused-argument
 from dataclasses import dataclass
 from typing import TYPE_CHECKING, List
 
-from pydantic import BaseModel
+from pydantic import BaseModel, Field
 
 from oteapi.plugins import StrategyFactory
 
 if TYPE_CHECKING:
-    from typing import Any, Dict, Optional
+    from typing import Any, Dict
 
     from oteapi.models import FilterConfig
 
 
 class CropDataModel(BaseModel):
-    crop: List[int]
+    """Configuration model for crop data."""
+
+    crop: List[int] = Field(..., description="List of image cropping details.")
 
 
 @dataclass
 @StrategyFactory.register(("filterType", "filter/crop"))
 class CropFilter:
+    """Strategy for cropping an image.
+
+    **Registers strategies**:
+
+    - `("filterType", "filter/crop")`
+
+    """
 
     filter_config: "FilterConfig"
 
-    def initialize(
-        self, session: "Optional[Dict[str, Any]]" = None
-    ) -> "Dict[str, Any]":
-        """Initialize strategy and return a dictionary"""
+    def initialize(self, **_) -> "Dict[str, Any]":
+        """Initialize strategy and return a dictionary."""
         return {"result": "collectionid"}
 
-    def get(self, session: "Optional[Dict[str, Any]]" = None) -> "Dict[str, Any]":
+    def get(self, **_) -> "Dict[str, Any]":
```

Comment:
General comment: I am not fully comfortable with this use of anonymous keyword arguments. The `get()` and several other methods takes one argument `session` as input. But it is not all implementations that uses it. I think it is better to be explicit about that.